pkg/modules/application_mobility.go

```diff
@@ -48,12 +48,16 @@ const (
 	ControllerImagePullPolicy = "<CONTROLLER_IMAGE_PULLPOLICY>"
 	//UseVolSnapshotManifest - filename of use volume snapshot manifest for app-mobility
 	UseVolSnapshotManifest = "velero-volumesnapshotlocation.yaml"
-	//CleanupCrdManifest - filename of Cleanup Crds manifest for app-mobility
+	// CleanupCrdManifest - filename of Cleanup Crds manifest for app-mobility
 	CleanupCrdManifest = "cleanupcrds.yaml"
-	//VeleroCrdManifest - filename of Velero crds manisfest for Velero feature
+	// VeleroCrdManifest - filename of Velero crds manisfest for Velero feature
 	VeleroCrdManifest = "velero-crds.yaml"
-	//VeleroAccessManifest - filename where velero access with its contents
+	// VeleroAccessManifest - filename where velero access with its contents
 	VeleroAccessManifest = "velero-secret.yaml"
+	// ResticCrdManifest - filename of restic manifest for app-mobility
```

Comment:
Where is this variable used?

Reply:
it might be from an older version where NodeAgentCrdManifest  was ResticCRDManifest:
see: https://github.com/dell/csm-operator/commit/b57367ed0d190d81266f67a8ff44bdc409adb195#diff-cb92caecc3ce72f1d9e89c59a02f703d1842410acaa9247fe2df7a1ad11bf9f3

 I don't think we need it

Reply:
This change was to just standardize the spacing between the comments. I am unsure where these are used.

Reply:
Ok, I think we forgot to remove it then, its not used anywhere in file
Can you remove it?

Reply:
Looked through, you are correct, this is not being used. Removing

Reply:
Fixed per commit f95b49cb8ba89ca11b4fea445bf2ce735ada56fc